### PR TITLE
Consistency for ways to use 2.0 API (common / current)

### PR DESCRIPTION
**Pull request by keedya**
- Able to switch on / by default in one place

### CODE REVIEW

**Comment by keedya** [comment]
@zyoung51 @brianparry

File: lib/services/http-service.js [hunk 1]
```diff
@@ -218,6 +219,11 @@ function httpServiceFactory(
             )
         );
 
+        //re-route common and current
+        var varsionPath = configuration.get('versionBase') || '1.1';
```
Comment (zyoung51) [inline lib/services/http-service.js]: versionPath instead of varsionPath?

**Comment by zyoung51** [comment]
Just a nitpick on the variable name. LGTM, +1

File: lib/services/http-service.js (hunk 1 above)
Comment (keedya) [inline lib/services/http-service.js]: Oh wow, totally missed that!! Thanks @zyoung51

**Comment by keedya** [comment]
@zyoung51 all set

**Comment by yyscamper** [comment]
:+1:

File: lib/services/http-service.js [hunk 2]
```diff
@@ -218,6 +219,11 @@ function httpServiceFactory(
             )
         );
 
+        //re-route common and current
+        var versionPath = configuration.get('versionBase') || '1.1';
```
Comment (brianparry) [inline lib/services/http-service.js]: @keedya I think we should add an assert or some other type of check to verify that `versionBase` is set to a valid value.

**Comment by brianparry** [comment]
:+1: One minor comment related to config validation, but I think that's more of a general problem outside the scope of this PR.

**Comment by brianparry** [comment]
test this please

**Pull request merged**